Build/GameInitOptionsScreen.cc

```diff
@@ -288,14 +277,7 @@ static void EnterGIOScreen()
 	{ // Check box to toggle difficulty settings
 		INT16 const x = GIO_DIF_SETTINGS_X + GIO_OFFSET_TO_TOGGLE_BOX;
 		INT16 const y = GIO_DIF_SETTINGS_Y - GIO_OFFSET_TO_TOGGLE_BOX_Y;
-		size_t def;
-		switch (gGameOptions.ubDifficultyLevel)
-		{
-			case DIF_LEVEL_EASY:   def = GIO_DIFF_EASY; break;
-			default:
-			case DIF_LEVEL_MEDIUM: def = GIO_DIFF_MED;  break;
-			case DIF_LEVEL_HARD:   def = GIO_DIFF_HARD; break;
-		}
+		size_t def = gGameOptions.ubDifficultyLevel - 1;
```

Comment:
Previously, 1-based difficulty level was converted to 0-based (1-0, 2-1, 3-2). We can achieve the same thing by just subtracting 1 from 1-based difficulty.